docs/Transferring-Roms.md

```diff
@@ -11,69 +11,68 @@ There are three main methods of transferring roms: via USB stick, via SFTP, and
-3. Select **Interfacing Options**
-4. Select **SSH**
-5. Choose **Yes**
-6. Select **Ok**
-7. Choose **Finish**
+1. Within the 'RetroPie' EmulationStation options menu, select **raspi-config**
+2. Select **Interface Options**
+3. Select **SSH**
+4. Choose **Yes**
+5. Select **Ok**
+6. Choose **Finish**
 
 There are many SFTP programs out there:
 
-- Windows: [WinSCP](https://winscp.net/eng/download.php)
+- Windows: 
+  - [WinSCP](https://winscp.net/eng/download.php) - For easy drag & drop file transfer
+  - [MobaXTerm](https://mobaxterm.mobatek.net/) - Feature-rich command line access and drag & drop file transfer
 - Mac: [Cyberduck](https://cyberduck.io/?l=en)
 
 ![ftp](https://cloud.githubusercontent.com/assets/10035308/9144892/68994618-3d0d-11e5-8db0-2991f9068115.png)
 
 **Connection settings:** 
 
 - Protocol: `SFTP`
-- IP address: To find the IP address of your RetroPie, go into RetroPie options from the main menu, and select the last option `Show IP address`. You can also find this information from the terminal on retropie in the bash info or with the command `ifconfig`
+- IP address/Host Name: To find the IP address of your RetroPie, go into the 'RetroPie' EmulationStation options menu and select **Show IP**. You can also find this information from the terminal on Retropie in the bash info or with the command `ifconfig`
 - Username: `pi` (default)
 - Password: `raspberry` (default)
 
-**Where to drop the files***
+**Where to drop the files:**
 
-Simply drop the files in the ~/RetroPie/roms/$CONSOLE folder, where $CONSOLE is the name of the target console, e.g. `snes` or `arcade`.
+Simply drop the files in the `~/RetroPie/roms/$CONSOLE` folder, where $CONSOLE is the name of the target console, e.g. `snes` or `arcade`.
 
 You can also log in as root if you wish to change more files than just the roms, but you first need to enable the root password by typing `sudo passwd root` into the terminal and choosing a new root password.
 
+For more information, check the [Raspberry Pi SSH documentation](https://www.raspberrypi.org/documentation/remote-access/ssh/).
+
 ### Samba-Shares
 
 [Samba](https://www.samba.org/samba/what_is_samba.html) is a software suite that allows you to access file systems over the network. Naturally both your PC and Pi will need to be connected to the same network via Ethernet or Wifi in order to successfully transfer your files. 
 
-- if on windows type `\\RETROPIE` into the computer folder. You can also replace RETROPIE with your Raspberry Pi's IP address
+- On Windows, type `\\RETROPIE` into the computer folder. You can also replace RETROPIE with your Raspberry Pi's IP address
 
 ![samba](https://cloud.githubusercontent.com/assets/10035308/9141308/edee8b52-3cf4-11e5-8bf3-73f8c27f99fb.png)
 
-- if on MAC OS X open finder, select "Go" menu and "Connect to Server". Type `smb://retropie` and hit "Connect".
-
+- On MAC OS X, open Finder, select "Go" menu and "Connect to Server". Type `smb://retropie` and hit "Connect".
```

Comment:
It's called `macOS` now, maybe add it as _Mac OS X/macOS_.

Reply:
Ok